orbit/pkg/packaging/wix/wix.go

```diff
@@ -7,30 +7,54 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
-	directoryReference = "ORBITROOT"
+	directoryReference   = "ORBITROOT"
+	linuxPathSeparator   = "/"
+	windowsPathSeparator = "\\"
 )
 
+// windowsJoin returns the result of replacing each slash ('/') character in
+// each path with a Windows separator character ('\') and joining them using
+// the Windows separator character.
+//
+// We can't use filepath.FromSlash because this func is run in a *nix
+// machine.
+func windowsJoin(paths ...string) string {
+	s := strings.Join(paths, windowsPathSeparator)
+	return strings.ReplaceAll(s, linuxPathSeparator, windowsPathSeparator)
+}
+
 // Heat runs the WiX Heat command on the provided directory.
 //
 // The Heat command creates XML fragments allowing WiX to include the entire
 // directory. See
 // https://wixtoolset.org/documentation/manual/v3/overview/heat.html.
-func Heat(path string) error {
-	cmd := exec.Command(
-		"docker", "run", "--rm", "--platform", "linux/amd64",
-		"--volume", path+":/wix", // mount volume
-		"fleetdm/wix:latest",  // image name
-		"heat", "dir", "root", // command in image
-		"-out", "heat.wxs",
+func Heat(path string, native bool) error {
+	var args []string
+
+	if !native {
+		args = append(
+			args,
+			"docker", "run", "--rm", "--platform", "linux/amd64",
+			"--volume", path+":"+path, // mount volume
```

Comment:
This will use the same `/tmp` path in both in the host and docker container, right?

Reply:
This is correct 👍 , the motivation is to have an uniform location for the build artifacts on native _and_ non-native builds.

Another alternative could be to use a symlink or copy the artifacts when `native` is `true`. If you have other ideas I'd be in to give it a try since I don't love the whole `windowsJoin` thing.